packages/web/src/common/store/cache/tracks/sagas.js

```diff
@@ -249,10 +249,7 @@ function* deleteTrackAsync(action) {
   const handle = yield select(getUserHandle)
 
   // Before deleting, check if the track is set as the artist pick & delete if so
-  const socials = yield call(
-    audiusBackendInstance.getCreatorSocialHandle,
-    handle
-  )
+  const socials = yield call(audiusBackendInstance.getSocialHandles, handle)
```

Comment:
For these js files I would just double check that the data is returned in the same format

Reply:
this fn is just renamed :) but yes get your point!!